ci/pipeline

```diff
@@ -30,12 +30,13 @@ def compileAndTest(logFileName: String): Unit = utils.stage("Compile and Test")
 
   def run(cmd: String *) = utils.runWithTimeout(30.minutes, logFileName)(cmd)
 
-  run("sbt", "clean", "test", "integration:test", "scapegoat")
+  //run("sbt", "clean", "test", "integration:test", "scapegoat")
```

Comment:
don't forget to un-comment

Reply:
Thanks, Mr. Obvious ^^.

Reply:
You are right though. I've added the `wip` tag.